### ISSUE TITLE
In the resumable model  how to call custom_objects.

### ISSUE THREAD

**Issue by Dingkuba** [dorukkarinca/keras-buoy#1]
resumableModel=ResumableModel(model,save_every_epochs=1,to_path='checkpoint.h5')
value error:Unknown loss function:dice_coef_loss
How do make this work?

**Comment by dorukkarinca** [comment]
You should do `model.compile(...)` and then do `resumableModel=ResumableModel(model, ...)` instead of `resumableModel.compile(...)`

**Comment by Dingkuba** [comment]
@dorukkarinca Actually i have  saved the checkpoint using the same procedure/guidelines as  mentioned ,it saves(no issue)   but whenever i try to load the checkpoint to resume my training where i left off its showing that error, even i  did the model.compile before resumableModel=resumableModel(..)  but this gives me error. showing unknown loss function as mentioned above.

**Comment by dorukkarinca** [comment]
Oh now I see. It appears there has been [an issue](https://github.com/keras-team/keras/issues/5916) about Keras with custom losses. Does [this solution](https://github.com/keras-team/keras/issues/5916#issuecomment-290344248) solve your problem?

The best solution would be to modify the internal call to load_model to accept a custom object as seen [here](https://github.com/keras-team/keras/issues/5916#issuecomment-300038263). I'll put this on the roadmap but I am a little backed up at the moment. Feel free to submit a PR if you do it first.

**Comment by Dingkuba** [comment]
@dorukkarinca that solution does not  solve it ,the best is to modify ur model.py where you pass def (model,steps_per_epoch ,path) i think there should be one to accept the custom_object in the function  because when i use the new_model = keras.models.load_model('bestmodel.hdf5',custom_objects={'dice_coef_loss':                   
dice_coef_loss,'dice_coef': dice_coef}) it doesnt give error so what i think is that a bit modification in ur  model.py or something would make that work as far as i know .tough i am newbie in github but i will try  to submit a PR

**Comment by dorukkarinca** [comment]
Looks like @ViiSkor solved this! https://github.com/ViiSkor/keras-buoy/commit/1a793413aa098ebac106baecf4d64bb516b4c5f1

I just asked her for a PR but in the meanwhile you may modify your local version like that.

**Comment by dorukkarinca** [comment]
PR https://github.com/dorukkarinca/keras-buoy/pull/2 solves this issue. Closing for now. Lmk if that works!
